packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -64,6 +64,23 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+MeetingsUtil.parseparseUserPreferences = (userPreferences) => {
```

Comment:
I don't see any unit tests for `parseparseUserPreferences()` or for `fetchUserPreferredWebexSite()`

Reply:
i actually test them from the parent function , we dont have test for the utils functions separately ,  

we should have added it but we dont have any currently can create one

Reply:
move move it back , i was using the register test 

`        it('on register makes sure following functions are called ', async () => {
          webex.canAuthorize = true;
          webex.meetings.registered = false;
          await webex.meetings.register();
          assert.called(webex.internal.device.register);
          assert.called(webex.internal.services.fetchLoginUserInformation);
          assert.called(webex.internal.services.fetchClientRegionInfo);
          assert.called(webex.internal.mercury.connect);
          assert.isTrue(webex.meetings.registered);
        });`